lib/vpr/configuration.rb

```diff
@@ -0,0 +1,17 @@
+require 'singleton'
+
+module Vpr
+  class Configuration
+    include Singleton
+
+    attr_accessor :remote
+
+    def self.remote=(upstream)
+      instance.remote = upstream.to_sym
+    end
+
+    def self. remote
```

Comment:
`def self.remote`

Reply:
My bad. But I decide to remove this method in favor to keep it simple and use the `remote` 
`attr_accessor`